jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/DatagramChannelEndPoint.java

```diff
@@ -46,6 +46,21 @@ public DatagramChannel getChannel()
         return (DatagramChannel)super.getChannel();
     }
 
+    @Override
+    public SocketAddress getRemoteSocketAddress()
```

Comment:
This should be moved to `SelectableChannelEndPoint` where `getLocalSocketAddress()` is.
And be removed also from `SocketChannelEndPoint`.

Reply:
You can't as the `getRemoteAddress` on a `DatagramChannel` is a different method than the one on `SocketChannel` and does not exist in `NetworkChannel`.

There is a reason it is as it is.